Remove standard libs from LD_RUN_PATH as per Debian policy.

diff -Naur --exclude=debian perl-5.8.7.orig/lib/ExtUtils/Liblist/Kid.pm perl-5.8.7/lib/ExtUtils/Liblist/Kid.pm
--- perl-5.8.7.orig/lib/ExtUtils/Liblist/Kid.pm	2004-04-04 23:31:31.000000000 +1000
+++ perl-5.8.7/lib/ExtUtils/Liblist/Kid.pm	2005-06-03 00:03:49.000000000 +1000
@@ -54,6 +54,9 @@
     my($pwd) = cwd(); # from Cwd.pm
     my($found) = 0;
 
+    # Debian-specific: don't use LD_RUN_PATH for standard dirs
+    $ld_run_path_seen{$_}++ for qw(/lib /usr/lib /usr/X11R6/lib);
+
     foreach $thislib (split ' ', $potential_libs){
 
 	# Handle possible linker path arguments.
